Rubberduck.Inspections/Concrete/UnreachableCaseInspection/UnreachableCaseInspectionFactoryFactory.cs

```diff
@@ -0,0 +1,46 @@
+﻿
+namespace Rubberduck.Inspections.Concrete.UnreachableCaseInspection
+{
+    public interface IUnreachableCaseInspectionFactoryFactory
```

Comment:
Yeah - I cringed a bit (OK... a lot) when I created this class   #toomuchofagoodthing.  

As far as the acronym/prefix, I kept the prefix of class names that would show up in the Inspection class similar to the Inspection itself (i.e., spelled-out).  The classes that are hiding behind my local interfaces I used the acronym.  So...If the FactorFactory gets injected to the Inspection's constructor (should it?)  - the Insepction's ctor signature will not show the acronym.

Pretty arbitrary...but that was my thinking at the time.  I don't see a lot of acronyms in the the other files, but I felt that the file and class name length was getting out of hand as I added support classes.

Reply:
I still think that we should avoid acronyms. If those are in fact support and thus aren't meant to be consumed by us directly, then move them off them into their own namespace (e.g. `Rubberduck.Inspections.Concrete.UnreachableCaseInspection.Support`). 

FWIW, I might consider writing it as `FactoryProvider` rather than `FactoryFactory`, and could be also made an auto-magic factory by the CW, I _think_. (my uncertainty is how CW will handle an automagic factory that returns more auto-magic factories. It might work but need to check.)